modules/serverless-functions-all-values-in-secret-to-env-variables.adoc

```diff
@@ -6,7 +6,7 @@
 [id="serverless-functions-all-values-in-secret-to-env-variables_{context}"]
 = Setting environment variables from all values defined in a secret
 
-You can use the following procedure to set an environment variable from all values defined in a secret.
+You can set an environment variable from all values defined in a secret. Values previously stored in a secret can then be immediately used in other parts of the `func.yaml` file.
```

Comment:
Similar comment here as above. The behavior is in the function at runtime, not in the `func.yaml` file per se.